contribute/backend/style-guide.md

```diff
@@ -69,6 +69,7 @@ Use [`t.Cleanup`](https://golang.org/pkg/testing/#T.Cleanup) to clean up resourc
 Optionally, we use [`mock.Mock`](https://github.com/stretchr/testify#mock-package) package to write mocks. This is
 useful when you expect different behaviours of the same function.
 
+
```

Comment:
This newline will be an issue if `prettier` is enforced in CI on this file.

```suggestion
```

Reply:
My original PR had "behaviours" to "behaviors." This was merged out in the merge from main.  and the newline was added.

Reply:
The writer's toolkit says, https://grafana.com/docs/writers-toolkit/, "We write our documentation using U.S. English." This entire PR was about maintaining consistency.

Reply:
> My original PR had "behaviours" to "behaviors." This was merged out in the merge from main. and the newline was added.
> The writer's toolkit says, grafana.com/docs/writers-toolkit, "We write our documentation using U.S. English." This entire PR was about maintaining consistency.

Ah, that's unfortunate. Correcting "behaviours" to "behaviors" would indeed be useful given our style but, since that has been merged away, this hunk isn't doing anything except introducing a `prettier` linting issue. 

I'll propose a suggestion in a separate comment.